include/domain_bridge/parse_domain_bridge_yaml_config.hpp

```diff
@@ -33,42 +33,46 @@ class YamlParsingError : public std::runtime_error
   {}
 };
 
-/// Create a DomainBridge from a YAML file.
+/// Create a DomainBridgeConfig object from a YAML file.
 /**
  * The YAML file may contain the following optional keys:
  *
  * - name: Name of the domain bridge
+ * - from_domain: The default 'from_domain' used for bridged topics.
+ *     If omitted, then each topic bridge must specify it's own value.
+ * - to_domain: The default 'to_domain' used for bridged topics
+ *     If omitted, then each topic bridge must specify it's own value.
  * - topics: A map of topic names to a map of topic bridge information
  *
- * For each topic bridge, the follow information is required, in the form of YAML keys:
+ * For each topic bridge, the follow keys are valid:
  *
- * - type: Message type to bridge
- * - from_domain: Subscribe to the topic on this domain ID
- * - to_domain: Publish to the topic on this domain ID
+ * - type: Message type to bridge (required)
+ * - from_domain: Subscribe to the topic on this domain ID (optional)
```

Comment:
I don't know if I'd put "required" or "optional" here since it depends on the context - whether there is a default from_domain and/or to_domain.

Reply:
Removed qualifiers in 609e2c9